C/other/concatenate_two_strings.c

```diff
@@ -0,0 +1,30 @@
+// The best way to concatenate two strings in C programming is by using the strcat() function.But,in this example, we will concatenate two strings manually.
```

Comment:
Few changes: 

- Add a beginner-friendly description at the top and make it concise and looks readable.